ui/library/src/components/BoundingBox.tsx

```diff
@@ -30,6 +32,7 @@ export const BoundingBox: React.FunctionComponent<Props> = ({
   const componentClassName = classNames(
     'reader__page-overlay__bounding-box',
     isHighlighted === true ? 'reader__page-overlay__bounding-box-highlighted' : '',
+    isPopoverVisible === true ? 'reader__page-overlay__bounding-box--selected' : '',
```

Comment:
Nit: `-selected` instead of `--selected` for consistency

Reply:
Actually where is this even being used? How does a popover being visible translate to something being selected? Would a class name like `reader__page-overlay__bounding-box-popover-visible` be more appropriate? Or should `isPopoverVisible` be updated to something like `isBoundingBoxSelected` because the popover isn't a concept the library even knows about?

Reply:
@carolinepaulic this being used in both the S2, component library side whenever a pop over is visible we want to change the selected color of it